themes/cardinal_service/pdb_components/opportunities_list/src/app.jsx

```diff
@@ -41,6 +41,7 @@ ReactDOM.render(
     wrapperAttributes={{className: "flex-12-of-12"}}
   >
     <div className="flex-12-of-12 sort-links">
+      <span style={{ fontWeight: 'bold' }}>Sort By:</span>
```

Comment:
Screen readers will pick this up and read "Sort By" twice because of the parts below. you should hide this from screen readers with `aria-hidden={true}`